### DISCUSSION TITLE
Mixing env variables between projects

### DISCUSSION THREAD

**Discussion by Cuboctaedro** [vercel/community#1050] [Help]
I have two projects (Next) using the same repo and the same branch but different environment variables, or to be more precise the same variables with different values for each project, as a form of multi-tenancy. 

Whenever I push to the repo and an atomatic build starts on all the projects `env` variable values sometimes to mix up and each project uses values of another. In most cases if I rerun the build process on each project separately through the UI then they get the correct variables. I have try using the `VERCEL_FORCE_NO_BUILD_CACHE` but it does not seem to help. Any ideas on how should I handle it?

**Comment by amyegan** [comment]
Hi @Cuboctaedro! Can you share a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) so I can get a better understanding of your setup? That will help me get a closer look at what is going wrong.
